src/main/java/org/fundacionjala/sfdc/pageobjects/Contacts/SFCNewModifyPage.java

```diff
@@ -0,0 +1,88 @@
+package org.fundacionjala.sfdc.pageobjects.Contacts;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.fundacionjala.sfdc.pageobjects.SalesForceConnection;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class SFCNewModifyPage extends SalesForceConnection {
+    private WebDriverWait wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)